spec/generators/data_migration/data_migration_generator_spec.rb

```diff
@@ -58,9 +62,56 @@
           'data_migration.rb', data_migrations_file_path
         )
 
+        expect(subject).not_to receive(:template)
+
         subject.create_data_migration
       end
     end
+
+    context 'when test generation is enabled' do
+      let(:rails_root) { 'dummy-app' }
+
+      before do
+        DataMigrate.config.data_migrations_path = 'db/data/'
+        DataMigrate.config.test_generator_enabled = true
+
+        allow(Rails).to receive(:root).and_return(Pathname.new(rails_root))
+      end
+
+      after(:all) do
+        FileUtils.rm_rf(Dir.pwd + '/db/data')
```

Comment:
Fun fact: Not setting `DataMigrate.config.data_migrations_path` and trying to use it to cleanup test files results it the spec trying to cleanup your root filesystem. 🔥 I had the fun time of finding that out the hard way